macos/install_all.sh

```diff
@@ -2,5 +2,5 @@
 source ../common/arch.sh
 
 for arch in $ARCHES; do
-brew install binutils-$arch.rb
-done
\ No newline at end of file
+brew upgrade binutils-$arch.rb
```

Comment:
Have you tested this? Someone pointed out that this might be wrong.

Reply:
Must have accidentally saved the install_all.sh when I was closing it out for the update_all.sh